HISTORY.rst

```diff
@@ -14,6 +14,7 @@ History
 "predict" in regression.py
 * Add replication of the Chen Xu's tutorial testing Jackknife+aB vs Jackknife+
 * Add Jackknife+-after-Bootstrap documentation
+* Add EnbPI method for Time Series Regression
```

Comment:
Maybe mention only the first implementation with abs(y-y_pred) and without beta ?

Reply:
J'ai rajouté une phrase à la fin pour dire que c'est une première étape